cypress.json

```
@@ -0,0 +1 @@
+{}
```

```suggestion
{
  "baseUrl": "http://localhost:3000",
  "integrationFolder": "cypress/e2e",
  "viewportHeight": 900,
  "viewportWidth": 400
}
```
Please follow the existing configuration that we have in https://github.com/kawalcovid19/wargabantuwarga.com/blob/main/cypress.json, except for the `projectId` attribute.